roles/basic_setup_rhel7/README.md

```diff
@@ -0,0 +1,78 @@
+theforeman.foreman.basic_setup_rhel7
+====================================
+
+This role is an opinionated reuse of other roles in the collection, which creates a basic configuration for everything Katello needs to register and patch existing RHEL7 clients.
+
+That includes uploading a subscription manifest an organization, enabling the rhel-7-server-rpms repository and creating a sync plan for it, optionally syncing it immediately, and creating an activation key <name> to use when registering RHEL7 clients.
+
+The subscription manifest will be retrieved from the specified path on the ansible target host; optionally, it can be fetched first from the RHSM portal using the provided login credentials and manifest UUID. It will be uploaded to the specified organization.
```

Comment:
```suggestion
The subscription manifest will be retrieved from the specified path on the Ansible target host; optionally, it can be fetched first from the RHSM portal using the provided login credentials and manifest UUID. It will be uploaded to the specified organization.
```